src/layouts/PageLayout/PageLayout.module.scss

```diff
@@ -0,0 +1,23 @@
+/* Copyright (c) Fortanix, Inc.
+|* This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of
+|* the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+@use '../../styling/defs.scss' as bk;
+
+@layer baklava.components {
+  .bk-page-layout {
+    @include bk.component-base(bk-page-layout);
+
+    display: flex;
+    flex-direction: column;
+    gap: bk.$spacing-7;
+
+    > [role="tablist"] {
```

Comment:
This could be fragile. Maybe add a container component, like:

```
<PageLayout>
  <PageLayout.Nav>
    <Tabs>...</Tabs>
  </PageLayout.Nav>
</PageLayout>
```

I'm calling it "Nav" here so that it could be used by other navigation components as well in the future.